example/generator/.gitignore

```diff
@@ -1 +1,3 @@
 odp_generator
+*.log
+*.trs
```

Comment:
I think this should be now in one of top-level .gitignore files.

Reply:
It isn't, at least in the master version on which this branch is based.